org.jacoco.core.test/src/org/jacoco/core/test/validation/ExceptionsTest.java

```diff
@@ -23,17 +23,6 @@
  */
 public class ExceptionsTest extends ValidationTestBase {
 
-	/**
-	 * https://bugs.openjdk.java.net/browse/JDK-8180660
-	 */
-	private static final boolean isJDK8u152;
-
-	static {
-		final Matcher m = Pattern.compile("1\\.8\\.0_(\\d++)(-ea)?")
```

Comment:
@Godin Deletion leads to unused imports warning for `java.util.regex.*`.

Reply:
@marchof fixed

Reply:
@Godin Thx! ✅